ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
+
+=== Packaging into a WAR deployment artifact
+You can also choose to package your application into a WAR deployment artifact for deployment on an Open Liberty server.
+Use Maven or Gradle to package your application into a WAR file so that it can be manually deployed on Open Liberty.
+
+Specify `war` as the packaging type in your `pom.xml` file (for Maven) or in your `build.gradle` file (for Gradle). Then, package your application with Maven or Gradle. The result is a WAR deployment artifact that you can deploy on Open Liberty.
+
+=== Packaging into a runnable JAR file
+If you want to package locally, you can package your application and an Open Liberty installation into a link:/docs/ref/general/#runnablejarfiles.html[runnable JAR file].
 The JAR file is portable and ready to deploy both the application and server together.
-Packaging locally can be useful for debugging and testing during development.
-
-With Maven, run the following goal that overrides the `include` option to specify a minimum runnable JAR file as the packaging type:
-----
-mvn liberty:package -Dinclude=minify,runnable
-----
-
-With Gradle, include the following `packageLiberty` block in the `libertyPackage` task to specify a minimum runnable JAR file as the packaging type:
-----
-packageLiberty {
-    packageName = "MyPackage"
-    include = "minify,runnable"
-}
-----
-
-Each example overrides the `include` property to package your application and runtime into a runnable JAR file.
-The `minify` option specifies that the runnable JAR file contains only what you need to run your application so the application is as lightweight as possible.
-
-== Containerize
-After you create a server package by packaging your application with the Open Liberty runtime, you can containerize it for deployment.
+This local packaging method can be useful for debugging and testing during development.
+
+To package into a runnable JAR file, specify `runnable` as the package type in the `include` parameter or attribute.
+You can also specify the `minify` option in the `include` property or attribute so that your runnable JAR file contains only what you need to run your application.
+
+== Containerization
+After you package your application into a server package with `usr` contents and build it with Maven or Gradle, you can containerize it for cloud deployment.
 Containerizing your server package bundles together the environment that your application needs to run.
 With Docker containers, you have the portability to run your application on any machine that is running Docker.
 
 A Docker image is a multi-layered file that serves as a template for a Docker container.
 The simplest way to containerize your server package is by using the latest link:https://hub.docker.com/_/open-liberty[Open Liberty Docker images], which include the most recent feature versions and link:/docs/ref/general/#security-vulnerabilities.html[security fixes].
```

Comment:
I think Arthur would probably prefer we point people at the UBI images. In either case I think this calls for some kind of documentation on the different images  and how to decide which ones to use.

Reply:
I updated the link. Yeah, I think that would be outside the scope of this topic, but we probably need to add something to the backlog.